hack/check-file-length.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+
+filename=$1
+if [ -z "$filename" ]; then
+    echo >&2 $0 expects a filename
+    exit 1
+fi
+
+max=$(grep lint_maximum_file_length "$filename" | awk '{print $3}')
+len=$(wc -l < "$filename")
+
+if [ -z "$max" ]; then
+    echo >&2 $filename: maximum length not found
```

Comment:
Maybe instead of `maximum length` use please `lint_maximum_file_length` var name itself?